sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/BlobServiceAsyncClient.java

```diff
@@ -384,16 +396,24 @@ public String getAccountUrl() {
 
     PagedFlux<TaggedBlobItem> findBlobsByTags(FindBlobsOptions options, Duration timeout) {
         throwOnAnonymousAccess();
-        return new PagedFlux<>(
-            () -> withContext(context -> this.findBlobsByTags(options, null, timeout, context)),
-            marker -> withContext(context -> this.findBlobsByTags(options, marker, timeout, context)));
+        StorageImplUtils.assertNotNull("options", options);
```

Comment:
Is there a reason why `FindBlobsOptions` can no longer be null? Or did this throw an NPE if null previously and is being triggered sooner?

Reply:
Correct. It'll validate when we call the inner overload of findBlobsByTags anyway, but since we will set the maxResults on the options here, I wanted to assert that it wasn't null just a step earlier.